lib/resources/example_json/soarian_examples_document_reference.rb

```diff
@@ -0,0 +1,169 @@
+module Cerner
+  module Resources
+
+    SOARIAN_DOCUMENT_REFERENCE_DOCREF_BUNDLE ||= {
+	
+    "resourceType": "Bundle",
+    "id": "3def23df-44bf-4006-ab1c-72d0c176b1b0",
+    "meta": {
+        "lastUpdated": "2017-06-05T14:18:31.429-04:00"
+    },
+    "extension": [
+        {
+            "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+            "valueExtension": {
+                "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "papi022"
+                    },
+                    {
+                        "url": "Message",
+                        "valueString": "Information provided is for documents created from 2016-03-01 through 2016-03-29 inclusive."
+                    }
+                ]
+            }
+        },
+        {
+            "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+            "valueExtension": {
+                "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "fsp001"
+                    },
+                    {
+                        "url": "Message",
+                        "valueString": "Date parameter values passed with request were: date=>=2016-03-01T14:15:00&date=<2016-03-30T14:15:00"
+                    }
+                ]
+            }
+        }
+    ],
+    "type": "searchset",
+    "link": [
+        {
+            "relation": "self",
+            "url": "base-url/bf615549-44cc-4899-bfa9-2cfd86bf358c/DocumentReference?date=%3E%3D2016-03-01T14%3A15%3A00&date=%3C2016-03-30T14%3A15%3A00&_format=json&patientId=FaresRoni3425778"
+        }
+    ],
+    "entry": [
+        {
+            "resource": {
+                "resourceType": "DocumentReference",
+                "masterIdentifier": {
+                    "system": "http://loinc.org",
```

Comment:
Yes. we are fixing it before the certification. As mentioned updated the example as

"masterIdentifier":
{
 "system": " urn:ietf:rfc:3986", 
"value": " urn:oid:2.16.840.1.113883.10.20.22.1.2"
 }